src/api/config/routes.rb

```diff
@@ -287,6 +287,10 @@ def self.public_or_about_path?(request)
       post 'project/remove_person/:project' => :remove_person, constraints: cons
       post 'project/remove_group/:project' => :remove_group, constraints: cons
       get 'project/monitor/:project' => :monitor, constraints: cons, as: 'project_monitor'
+      # compat route
+      get 'project/monitor', to: redirect { |_path_parameters, request|
+        "/project/monitor/#{request.query_parameters[:project]}?#{request.query_parameters.except(:project).to_param}"
```

Comment:
where is this route actually used? What about stop using it instead of redirecting? :eyes:

Reply:
What about not breaking people's bookmarks for no reason?

Reply:
@darix 

> What about not breaking people's bookmarks for no reason?

This comment was for @eduardoj, not for you. Fixing the Bookmarks doesn't means we have to do it in this way. Sorry that this broke for you, but this is IMO not the correct solution. We can come up with a fast solution if this is a priority bug (BTW, where is the bug report?), but this shouldn't stay like that.

Reply:
I got wrong was this what about